api-get-object-file.go

```diff
@@ -78,8 +78,13 @@ func (c Client) FGetObject(bucketName, objectName, filePath string) error {
 		return err
 	}
 
+	// Initialize get object conditions and set the appropriate
+	// range offsets to read from.
+	reqHeaders := NewGetReqHeaders()
+	reqHeaders.SetRange(st.Size(), 0)
```

Comment:
As per, `case start > 0 && end == 0:` in `SetRange(...)` method  this `reqHeaders` will request bytes of object starting from `st.Size()` to EOF, which is not intended. Shouldn't it be `SetRange(0, -st.Size())`?